### DISCUSSION TITLE
How to transcribe (original language) AND translate?

### DISCUSSION THREAD

**Discussion by tk-master** [openai/whisper#606] [General]
Hello,

Currently I see the options --task transcribe and --task translate but what if I want both?
I want to keep the original language transcipt and the translated one.

Do I really have to run the script twice for this? seems like a big waste of time.

**Comment by wwbnjsace** [comment]
i also want to know the answer, i want to get the asr result and the tranlations result

**Comment by mikkovedru** [comment]
I would also like to be able to do both.
